cli/src/pcluster/validators/slurm_settings_validator.py

```diff
@@ -114,3 +115,20 @@ def _validate(self):
                 FailureLevel.WARNING,
             )
             CustomSlurmSettingsWarning.signaled = True
+
+
+class CustomSlurmSettingsIncludeFileOnlyValidator(Validator):
+    """
+    Custom Slurm Settings Include File Only validator.
+
+    This validator returns an error if the CustomSlurmSettingsIncludeFile configuration parameter
+    is used together with the CustomSlurmSettings under SlurmSettings.
+    """
+
+    def _validate(self, custom_settings: List[Dict], include_file_url: str):
+        if custom_settings and include_file_url:
+            self._add_failure(
+                "CustomSlurmsettings and CustomSlurmSettingsIncludeFile cannot be used together "
+                "under SlurmSettings.",
```

Comment:
I know they can only be specified in this section, however I'd remove the last part of the sentence `under SlurmSettings` ...since they cannot be used together anywhere and this message can be misleading.

Reply:
Actually they can use the `CustomSlurmSettings` at Queue and ComputeResource level while still using `CustomSlurmSettingsIncludeFile`, since they are in two different parts of the Slurm configuration managed by ParallelCluster.